model/schema/gene.yaml

```diff
@@ -111,14 +143,48 @@ classes:
       - split_from
       - split_into
 
-
 ## ------------
 ## SLOTS
 ## ------------
 
+slots:
+
+  single_gene:
+    domain: Gene
```

Comment:
this should be "range: Gene" right? with no specification of domain?

Reply:
Good catch. Fixed.